docs/deployment-guide.md

```diff
@@ -0,0 +1,396 @@
+## WonderTix Deployment Guide
+
+Created by Jacob Bentley (jacobpbentley@gmail.com)
+
+![alt text](doc-images/image-11.png)
+
+### Introduction
+
+This document provides a walkthrough of the WonderTix deployment on Google Cloud Platform (GCP). Its purpose is to describe each of the main components and provide some instructions for common or important tasks. Where appropriate, additional details can be found in separate, linked documents.
+
+### The Big Three
+
+The three most important services in our GCP architecture are Cloud Build, Cloud Run, and Cloud SQL. We’ll go through each of them in turn, or you can click one of the above links to jump straight to that section.
+
+### Cloud Build
+
+#### Triggers and workflows
+
+Cloud Build is the link between our GitHub repo and the rest of GCP. It’s how our code gets packaged up into a deployable form and sent over to Cloud Run, which does the actual deploying. It does this primarily through the use of triggers (pictured below).
+
+![alt text](doc-images/doc-images/image-12.png)
+
+Each of our triggers is pointed at a given workflow. The conventional name for these workflow files is cloudbuild.yaml. You’ll find several of these workflows sprinkled around the GitHub repo, but the most important three are located at the root level, in the client directory, and in the server directory.
+
+- cloudbuild.yaml (root): runs our main test-and-deploy pipeline, which builds both the client and server, runs E2E tests on them, and then deploys them.
+- client/cloudbuild.yaml: builds and deploys the client.
+- server/cloudbuild.yaml: builds and deploys the server.
+
+When you run any of the triggers pictured above, you run the corresponding workflow in the Build configuration column. It’s important to note that these triggers and workflows aren’t necessarily limited to building: they can be used to do any number of tasks depending on context. The main thing is that each trigger is used to run a particular workflow, but we’ll get more into workflows in a bit. First let’s talk about the actual building.
+
+![alt text](doc-images/doc-images/image-13.png)
```

Comment:
This path should just be `doc-images/image-13.png`.

Reply:
https://github.com/WonderTix/WonderTix/pull/635/commits/75cd7ffeb075c08404495a127fbdc813b08b8ca4